charts/newrelic-infrastructure/templates/clusterrole.yaml

```diff
@@ -18,6 +18,7 @@ rules:
       - "services"
       - "nodes"
       - "namespaces"
+      - "pods" # Needed for our Windows integration to be able to access KSM on a Linux node.
```

Comment:
@RagalahariP From v2 to v3, the pods resource is removed for some reason as you mentioned during the meeting.
The reason may be reducing the security surface.

What I mean during the meeting is if we can adding it back for windows only, that will be better

Reply:
@xqi-nr after you left the meeting we agreed that this comment is a good compromise. @RagalahariP will contact @paologallinaharbur to request his review of this line change since he was involved in the project to upgrade V2 to V3 and may have historical context to validate this change

Reply:
Hello! Yes, I'll take a look!

In general, we have never added support for Windows due to the lack of a canary environment, tooling, and proper pipelines to build the images. 
Consider to specify clearly that this is experimental and might change in the future.

Reply:
By the way, answering this particular point, it was a permission not needed anymore and therefore it was dropped. 
You can reintroduce it without any issues, however if you want to ask for as few permission as possible you can add this specific line behind a "if windows.enabled=true" instead of a comment